### DISCUSSION TITLE
Disallow Foreign Key Arrays in the UI

### DISCUSSION THREAD

**Discussion by ptrtht** [supabase/supabase#20870] [Feature Requests]
[Array of foreign keys](https://commitfest.postgresql.org/19/1252/) are not supported by supabase; and are not planned as I see.

As in  #535 the suggestion method is to create junction tables instead.

In the FE UI after I define a foreign key relation, I would like the define as array option to be disabled with some help text added. See screenshot.

![Screenshot 2024-01-30 at 13 04 20](https://github.com/supabase/supabase/assets/47393586/b4dbf177-af9a-4cc3-b412-1592fa11ffd6)

This is a quick and easy task; and would save some dramatic time!

**Comment by yamz8** [comment]
any updates about that?
